src/screens/UserManagement/UserRevamp/NewUserInvitationForm.res

```diff
@@ -0,0 +1,202 @@
+
+  let getMemberAcessBasedOnRole = async _ => {
+    try {
+      setScreenState(_ => PageLoaderWrapper.Loading)
+      let url = getURL(
+        ~entityName=USER_MANAGEMENT,
+        ~userRoleTypes=ROLE_ID,
+        ~id=roleTypeValue,
+        ~methodType=Get,
+        ~queryParamerters=Some("groups=true"),
+      )
+      let res = await fetchDetails(url)
+      setRoleDict(prevDict => {
+        prevDict->Dict.set(roleTypeValue->Option.getOr(""), res)
+        prevDict
+      })
+
+      setScreenState(_ => PageLoaderWrapper.Success)
+    } catch {
+    | Exn.Error(e) => {
+        let err = Exn.message(e)->Option.getOr("Failed to Fetch!")
+        setScreenState(_ => PageLoaderWrapper.Error(err))
+      }
+    }
+  }
+
+  React.useEffect(() => {
+    if roleTypeValue->Option.isSome {
+      getMemberAcessBasedOnRole()->ignore
+    }
+    None
+  }, [roleTypeValue])
+
+  let onClickDropDownApi = async () => {
+    try {
+      setDropDownLoaderState(_ => DropdownWithLoading.Loading)
+
+      let url = getURL(
+        ~entityName=USERS,
+        ~userType=#LIST_ROLES_FOR_INVITE,
+        ~methodType=Get,
+        ~queryParamerters=Some(`entity_type=${(userEntity :> string)->String.toLowerCase} `),
+      )
+      let result = await fetchDetails(url)
+      setOptions(_ => result->makeSelectBoxOptions)
+      if result->getArrayFromJson([])->Array.length > 0 {
+        setDropDownLoaderState(_ => DropdownWithLoading.Success)
+      } else {
+        setDropDownLoaderState(_ => DropdownWithLoading.NoData)
+      }
+    } catch {
+    | _ => setDropDownLoaderState(_ => DropdownWithLoading.NoData)
+    }
+  }
+
+  <div className="flex flex-col">
+    <div className="grid grid-cols-6 gap-6 items-end p-6 !pb-10 border-b">
+      <div className="col-span-5 w-full">
+        <FormRenderer.FieldRenderer
+          field=inviteEmail labelClass="!text-black !text-base !-ml-[0.5px]"
+        />
+      </div>
+      <div className="col-span-1 w-full p-1">
+        <FormRenderer.SubmitButton
+          text={"Send Invite"}
+          loadingText="Loading..."
+          buttonSize={Small}
+          customSumbitButtonStyle="w-full !h-12"
+          tooltipForWidthClass="w-full"
+        />
+      </div>
+    </div>
+    <div className="grid grid-cols-5">
+      <div className="col-span-2 border-r p-6  flex flex-col gap-2">
+        <OrganisationSelection />
+        <MerchantSelection />
+        <ProfileSelection />
+        <DropdownWithLoading
+          options onClickDropDownApi formKey="role_id" dropDownLoaderState isRequired=true
+        />
+        <NoteComponent />
+        <FormValuesSpy />
+      </div>
+      <div className="p-6 flex flex-col gap-2 col-span-3">
+        {switch roleTypeValue {
+        | Some(role) =>
+          <>
+            <p className={`${p1MediumTextClass} !font-semibold py-2`}>
+              {`Role Description - '${role}'`->React.string}
```

Comment:
use LogicUtils title for showing it in better way
<img width="403" alt="image" src="https://github.com/user-attachments/assets/5c953622-fa22-4bf0-b1e4-4e4a8550f207">